src/qt/sendcoinsdialog.cpp

```diff
@@ -324,16 +324,22 @@ bool SendCoinsDialog::PrepareSendText(QString& question_string, QString& informa
         formatted.append(recipientElement);
     }
 
-    if (model->wallet().privateKeysDisabled() && !model->wallet().hasExternalSigner()) {
-        question_string.append(tr("Do you want to draft this transaction?"));
-    } else {
-        question_string.append(tr("Are you sure you want to send?"));
-    }
-
     question_string.append("<br /><span style='font-size:10pt;'>");
+    /*: Message displayed when attempting to create a transaction. Cautionary text to prompt the user to verify
+        that the displayed transaction details represent the transaction the user intends to create. */
+    question_string.append(tr("Do you want to create this transaction?"));
```

Comment:
Though the line is correctly displayed in the GUI, there is a lack of space after it, which seems odd.

**Screenshot (PR):**
Note: the slight white highlighting is done to emphasize and was not present while testing.
![Correction](https://user-images.githubusercontent.com/85434418/135666201-c3bbb8db-8006-4a90-ba96-312267bbdaf0.png)

Either a space should be added after the statement so that the spacing is corrected. Or it should be displayed as a header text as is done on the master branch:

**Screenshot (Master):**
![Master](https://user-images.githubusercontent.com/85434418/135666458-8289ac38-8059-4d6c-bf25-cf9d7f52e22e.png)

Reply:
Fixed